### PR TITLE
[IMP] theme_anelusia, *: adapt `s_website_form_overlay`

### PR DESCRIPTION
**Pull request by Brieuc-brd**
*: theme_aviato, theme_beauty, theme_bewise, theme_bistro, theme_bookstore, theme_buzzy, theme_clean, theme_cobalt, theme_enark, theme_graphene, theme_kea, theme_kiddo, theme_monglia, theme_nano, theme_notes, theme_odoo_experts, theme_orchid, theme_paptic, theme_real_estate, theme_treehouse, theme_vehicle, theme_yes, theme_zap

This commit adapts customizations for `s_website_form_overlay`.

task-4138795
Part of task-4077427

Requires:
- https://github.com/odoo/odoo/pull/183568

### CODE REVIEW

File: theme_avantgarde/views/images_library.xml [hunk 1]
```diff
@@ -477,5 +477,10 @@ Check in images.scss, primary_variables.scss, main.scss and theme_common's mixin
     <field name="name">website.s_showcase_default_image</field>
     <field name="url">/theme_avantgarde/static/src/img/pictures/bg_image_07.jpg</field>
 </record>
+<record id="s_website_form_overlay_default_image" model="theme.ir.attachment">
+    <field name="key">website.s_website_form_overlay_default_image</field>
+    <field name="name">website.s_website_form_overlay_default_image</field>
+    <field name="url">/theme_avantgarde/static/src/img/pictures/bg_image_08.jpg</field>
```
Comment (mano-odoo) [inline theme_avantgarde/views/images_library.xml:483]: It appears pixellated maybe something more abstract such as `s_pricelist_boxed_default_background.jpg` ?

File: theme_bewise/views/image_content.xml [hunk 2]
```diff
@@ -402,5 +402,10 @@
     <field name="name">website.s_three_columns_default_image_3</field>
     <field name="url">/theme_bewise/static/src/img/content/library_image_10.jpg</field>
 </record>
+<record id="s_website_form_overlay_default_image" model="theme.ir.attachment">
+    <field name="key">website.s_website_form_overlay_default_image</field>
+    <field name="name">website.s_website_form_overlay_default_image</field>
+    <field name="url">/theme_bewise/static/src/img/backgrounds/college_bg.jpg</field>
```
Comment (mano-odoo) [inline theme_bewise/views/image_content.xml:408]: Maybe another image `library_image_08` ? I don't know it's maybe ok

**Review by mano-odoo — CHANGES_REQUESTED** [review]
Thanks for your work, just a few remarks and nitpicks for you

File: theme_aviato/views/images_library.xml [hunk 3]
```diff
@@ -382,4 +382,11 @@
     <field name="url">/theme_aviato/static/src/img/content/s_empowerment_default_image.jpg</field>
 </record>
 
+<!-- Website Form Overlay -->
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply theme_aviato/views/images_library.xml:385]: Indeed, some themes add comments for each record, others don't. I've done this on purpose to maintain consistency with each theme.

File: theme_orchid/views/images.xml [hunk 4]
```diff
@@ -385,5 +385,10 @@ Check in theme_loftspace's primary_variables.scss and theme.scss -->
     <field name="name">website.s_empowerment_default_image</field>
     <field name="url">/theme_orchid/static/src/img/snippets/s_empowerment_default_image.jpg</field>
 </record>
+<record id="s_website_form_overlay_default_image" model="theme.ir.attachment">
+    <field name="key">website.s_website_form_overlay_default_image</field>
+    <field name="name">website.s_website_form_overlay_default_image</field>
+    <field name="url">/theme_orchid/static/src/img/snippets/s_parallax.jpg</field>
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply theme_orchid/views/images.xml]: Colors of `s_quotes_carousel_2` doesn't really match with the theme colors :/
I traded with `s_parallax.jpg`. 👍

[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply theme_orchid/views/images.xml]: Colors of `s_quotes_carousel_2.jpg` doesn't really match with the theme colors :/
I traded with `s_product_catalog_default_image.jpg` 👍

**Comment by Brieuc-brd** [comment]
Thanks for your review @mano-odoo !
Your image suggestions are nice ! I've updated the branch, could you have a look ?
Thanks in advance !

File: theme_orchid/views/images.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (mano-odoo) [reply theme_orchid/views/images.xml]: Indeed looks super cool!

**Review by mano-odoo — APPROVED** [review]
LGTM! Thanks for updating the images 😄

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**